src/blocks/HeaderBlockBrandPivot/index.tsx

```diff
@@ -0,0 +1,366 @@
+  typeof window !== 'undefined' && window.scrollY > 20
+
+const Wrapper = styled('div')<{ inverse: boolean; open: boolean }>(
+  ({ inverse, open }) => ({
+    position: 'fixed',
+    top: 0,
+    left: 0,
+    right: 0,
+    zIndex: 100,
+    color: inverse ? colorsV3.white : colorsV3.gray900,
+    transition: 'color 300ms',
+
+    [TABLET_BP_DOWN]: {
+      bottom: open ? 0 : undefined,
+    },
+  }),
+)
+const Filler = styled('div')({
+  height: WRAPPER_HEIGHT,
+})
+const HeaderBackgroundFiller = styled('div')<{ transparent: boolean }>(
+  ({ transparent }) => ({
+    position: 'absolute',
+    top: 0,
+    left: 0,
+    right: 0,
+    zIndex: -1,
+    height: WRAPPER_HEIGHT,
+    backgroundColor: colorsV3.white,
+    opacity: transparent ? 0 : 1,
+    transition: 'opacity 300ms',
+  }),
+)
+
+const InnerHeaderWrapper = styled('div')({
+  display: 'flex',
+  justifyContent: 'space-between',
+  alignItems: 'center',
+  width: '100%',
+  height: WRAPPER_HEIGHT,
+  padding: HEADER_VERTICAL_PADDING + ' 0',
+})
+const Menu = styled('ul')<{ open: boolean }>(({ open }) => ({
+  display: 'flex',
+  justifyContent: 'space-between',
+  alignItems: 'center',
+  margin: 0,
+  padding: 0,
+  listStyle: 'none',
+
+  [TABLET_BP_DOWN]: {
+    position: 'absolute',
+    display: 'block',
+    zIndex: 101,
+    width: '80%',
+    height: '100vh',
+    top: 0,
+    bottom: 0,
+    right: open ? '20%' : '100%',
+    paddingTop: `calc(${WRAPPER_HEIGHT} + ${HEADER_VERTICAL_PADDING})`,
+    fontSize: 18,
+    background: colorsV3.white,
+    transition: `right ${TOGGLE_TRANSITION_TIME}ms`,
+    color: colorsV3.gray900,
+    overflow: 'scroll',
+    WebkitOverflowScrolling: 'touch',
+  },
+}))
+
+const LogoLink = styled('a')({
+  display: 'inline-flex',
+  paddingTop: 3, // fix to push down logo a little or it looks unbalanced
+  color: 'inherit',
+  marginLeft: '1rem',
+})
+
+const ButtonWrapper = styled('div')({
+  paddingLeft: '2rem',
+
+  [TABLET_BP_DOWN]: {
+    paddingTop: '1.5rem',
+    paddingLeft: '1rem',
+    paddingBottom: HEADER_VERTICAL_PADDING,
+  },
+})
+
+const RightContainer = styled('div')({
+  display: 'flex',
+})
+
+const MobileHeaderLink = styled(ButtonLinkBrandPivot)({
+  [TABLET_BP_UP]: {
+    display: 'none',
+  },
+})
+
+interface HeaderBlockProps extends BaseBlockProps {
+  is_transparent: boolean
+  inverse_colors: boolean
+  override_cta_link?: LinkComponent | null
+  override_cta_label?: string | null
+  cta_color?: MinimalColorComponent
+  cta_style?: ButtonStyleType
+  override_mobile_header_cta_label?: string | null
+  override_mobile_header_cta_link?: LinkComponent | null
+  mobile_header_cta_color?: MinimalColorComponent
+  mobile_header_cta_style?: ButtonStyleType
+}
+
+enum InverseColors {
+  DEFAULT = 'standard-inverse',
+  INVERSE = 'standard',
+}
+
+export const Header: React.FC<{ story: GlobalStory } & HeaderBlockProps> = (
+  props,
+) => {
+  const [isBelowThreshold, setScrollPosition] = React.useState<boolean>(
```

Comment:
I think the setter here should match the state, so `setIsBelowThreshold`

Reply:
Word!